api/app/src/routes/medical/link.ts

```diff
@@ -0,0 +1,96 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import { asyncHandler, getCxIdOrFail, getPatientIdFromQueryOrFail } from "../util";
+const router = Router();
+import status from "http-status";
+import { getPatient } from "../../command/medical/patient/get-patient";
+
+/** ---------------------------------------------------------------------------
+ * POST /link
+ *
+ * Creates link to the specified entity.
+ *
+ * @return  {Link}  The created link.
+ */
+router.post(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getCxIdOrFail(req);
+    const patientId = getPatientIdFromQueryOrFail(req);
+    const patient = await getPatient({ id: patientId, cxId });
+    // get linkSource from query params -> this should be "CommonWell"
+    console.log(patient);
+
+    // CommonWell.patientLink();
+    // store the following data in our DB in the patient:
+    //    - link ID
+    //    - person ID
+    // can put this into a JSONB column `link_data` with structure:
+    //    {["CommonWell"]: {cw_link_id, cw_person_id}}
+
+    return res.status(status.OK).json({});
+  })
+);
+
+/** ---------------------------------------------------------------------------
+ * DELETE /link
+ *
+ * Removes the specified HIE link from the specified patient.
+ *
+ * @return  200
+ */
+router.delete(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getCxIdOrFail(req);
+    const patientId = getPatientIdFromQueryOrFail(req);
+    const patient = await getPatient({ id: patientId, cxId });
+    // get linkSource from query params -> this should be "CommonWell"
```

Comment:
ditto link/reference param

Reply:
Same thing here - this param is referenced in the FE call: https://github.com/metriport/metriport-internal/blob/4c6342e4f9be84674858aa882d9a30fbb04d8ea0/dash/app/src/api/medical/link.ts#L37-L63

@Orta21  FYI